sdk/python/kfp/compiler/compiler.py

```diff
@@ -53,6 +53,7 @@ def compile(
         pipeline_name: Optional[str] = None,
         pipeline_parameters: Optional[Dict[str, Any]] = None,
         type_check: bool = True,
+        k8s_manifests_format: bool = False,
```

Comment:
I think let's be explicit here since not all users may know the abbreviation k8s.
```suggestion
        kubernetes_manifest_format: bool = False,
```